packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -742,13 +741,14 @@ class _RenderDecoration extends RenderBox with SlottedContainerRenderObjectMixin
   RenderBox? get suffix => childForSlot(_DecorationSlot.suffix);
   RenderBox? get prefixIcon => childForSlot(_DecorationSlot.prefixIcon);
   RenderBox? get suffixIcon => childForSlot(_DecorationSlot.suffixIcon);
-  RenderBox? get helperError => childForSlot(_DecorationSlot.helperError);
+  RenderBox get helperError => childForSlot(_DecorationSlot.helperError)!;
```

Comment:
Why is this no longer nullable? Is this like the first one in a greater planned refactor?

Reply:
The `_layout` method assumes it's non-null so I think this makes the assumption more explicit.